controllers/topicsController.js

```diff
@@ -65,3 +67,13 @@ exports.postComment = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.patchArticle = (req, res, next) =>{
+  const {article_id} = req.params
+  const {inc_votes} = req.body
+  Promise.all([getArticleById(article_id), patchArticleById(article_id, inc_votes)])
```

Comment:
not sure why we need the Promise.all. You can returning * from the insert to see what was updated. Might save an extra query to the db